pkgs/flox-cli/default.nix

```diff
@@ -28,7 +28,7 @@
   pkgsFor,
   pre-commit-check,
   flox-pkgdb,
-  flox-env-builder,
+  longVersion ? false,
```

Comment:
`longVersion` can be removed, we're not using it anymore.

Reply:
I dropped it. Thanks